TAF_EasyRest/TestFramework/PageComponents/NavigationMenuPageComponents/NavigationMenuPageComponent.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace TestFramework.PageComponents.NavigationMenuComponents
```

Comment:
![image](https://user-images.githubusercontent.com/14205615/181253751-e57f74d0-d16b-4953-b2aa-b0d50d8becdd.png)
This must be the same name.

Reply:
changed